libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/IndividualFeatureImportanceView/IndividualFeatureImportanceView.styles.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  getTheme,
+  IStackTokens,
+  IStyle,
+  mergeStyleSets,
+  IProcessedStyleSet
+} from "@fluentui/react";
+
+export const verticalComponentTokens: IStackTokens = { padding: "l1" };
+export const horizontalComponentTokens: IStackTokens = {
+  childrenGap: "l1",
+  padding: `0px 0px 0px 25px`
+};
+
+export interface IIndividualFeatureImportanceViewStyles {
+  boldText: IStyle;
+}
+
+export const individualFeatureImportanceStyles: () => IProcessedStyleSet<IIndividualFeatureImportanceViewStyles> =
+  () => {
+    const theme = getTheme();
+    return mergeStyleSets<IIndividualFeatureImportanceViewStyles>({
+      boldText: {
+        fontSize: theme.fonts.large.fontSize,
```

Comment:
curious why is theme needed for fontsize

Reply:
When using Label instead of Text, the default font size is different.  Here I make it the same.  Ke suggested I use Label instead of Text for the bold elements instead of changing the fontweight.